suites/pacific/rgw/tier-2_rgw_regression.yaml

```diff
@@ -343,6 +343,16 @@ tests:
         config-file-name: test_bucket_lifecycle_config_versioning.yaml
         timeout: 300
 
+  - test:
+      name: Bucket Lifecycle Configuration Tests
+      desc: Test lifecycle with version enabled bucket containing only one object version
+      polarion-id: CEPH-11187
+      module: sanity_rgw.py
+      config:
+        script-name: test_bucket_lifecycle_object_expiration.py
+        config-file-name: test_lc_current_version_object_expiration.yaml
+        timeout: 300
+
```

Comment:
Can avoid adding this test case in this suite?

Reply:
done.